.semaphore/calculator_simple-macos.yml

```diff
@@ -0,0 +1,28 @@
+version: v1.0
+name: Apple/Mojave Based Pipeline
+agent:
+  machine:
+    type: a1-standard-4
+    os_image: macos-mojave-xcode11
+blocks:
+  - name: Setup
+    task:
+      jobs:
+        - name: brew_installs
+          commands:
+            - brew install gcc-9
+            - brew install gtk+3
+            - brew install gtkmm3
+  - name: Grab Source
+    task:
+      jobs:
+        - name: grab
+          commands:
+            - checkout
+  - name: Build
+    task:
+      jobs:
+        - name: build
+          commands:
+            - cd calculator_simple/build/macos/
+            - gmake
```

Comment:
We should delete this file until we figure out how to setup the file architecture for SemaphoreCI.

Reply:
Deleted with 028e41e